.travis.yml

```diff
@@ -19,10 +19,10 @@ jobs:
         - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
 
     - stage: tag-new-release-if-applicable
-      if: branch = master
+      if: branch = master AND type != pull_request
```

Comment:
I noticed that PR builds are running this check, which could easily cause weird problems. This behavior is really surprising to me...

Reply:
Confirmed that this is fixed on this branch

![screen shot 2018-02-23 at 10 39 45](https://user-images.githubusercontent.com/630449/36608062-e55d982e-1885-11e8-8b46-6b726e3f662d.png)

Reply:
I also filed https://github.com/travis-ci/travis-ci/issues/9260 to see what the travis folks think about it.